### PR TITLE
Update covenant

### PR DESCRIPTION
**Pull request by Art3miX**
Modify update covenant migration messages.

### CODE REVIEW

File: contracts/native-router/src/contract.rs [hunk 1]
```diff
@@ -167,23 +167,29 @@ fn try_route_balances(deps: DepsMut, env: Env) -> Result<Response, ContractError
             // neutron fees for
             let count = Uint128::from(1 + denom_balances.len() as u128);
 
-            for mut coin in denom_balances {
-                // if we distribute neutron, we need to keep a
-                // reserve for ibc gas costs.
-                // this is safe because we pass target denoms.
-                if coin.denom == "untrn" {
+            for coin in denom_balances {
+                // non-neutron coins get distributed entirely
```
Comment (bekauz) [inline contracts/native-router/src/contract.rs:171]: any reason for bringing this back? or could it be an old rebase?

Comment (Art3miX) [reply contracts/native-router/src/contract.rs]: My bad, fixed.

File: contracts/ibc-forwarder/src/msg.rs [hunk 2]
```diff
@@ -119,6 +119,8 @@ pub enum MigrateMsg {
         clock_addr: Option<String>,
         next_contract: Option<String>,
         remote_chain_info: Option<RemoteChainInfo>,
+        // TODO: allow updating the transfer amount?
+        // transfer_amount: Option<Uint128>,
```
Comment (bekauz) [inline contracts/ibc-forwarder/src/msg.rs:123]: I think this is reasonable but somehow should be done by also updating the rest of covenant fields that would be impacted

File: contracts/two-party-pol-holder/src/msg.rs [hunk 3]
```diff
@@ -466,6 +466,8 @@ pub enum MigrateMsg {
         pool_address: Option<String>,
         ragequit_config: Box<Option<RagequitConfig>>,
         covenant_config: Box<Option<TwoPartyPolCovenantConfig>>,
+        // TODO: Do we want to allow this?
```
Comment (bekauz) [inline contracts/two-party-pol-holder/src/msg.rs:469]: good question.. I think yes? If both parties agree, it's changing the covenant terms just as any other update

File: interchaintest/go.mod [hunk 4]
```diff
@@ -12,7 +12,7 @@ require (
 )
 
 require (
-	cosmossdk.io/api v0.2.6 // indirect
+	cosmossdk.io/api v0.2.6 // indirect; indirectq
```
Comment (bekauz) [inline interchaintest/go.mod:15]: rebase leftover

**Review by bekauz — COMMENTED** [review]
everything looks good, left comments regarding the questions

File: contracts/ibc-forwarder/src/msg.rs (hunk 2 above)
Comment (bekauz) [reply contracts/ibc-forwarder/src/msg.rs:123]: lets make this updatable

**Pull request merged**